### ISSUE TITLE
screen flic 

### ISSUE THREAD

**Issue by RSerejo** [Leayal/SnowBreakLauncher-Dotnet#4]
https://youtu.be/f0_6NGh_HZ4

problem to login on game.

![image](https://github.com/Leayal/SnowBreakLauncher-Dotnet/assets/43152685/5b019776-0c0f-4e7f-bb98-cd276121661c)

**Comment by Leayal** [comment]
Can you try launching the game directly by opening the game directory, and follow the `game/Game/Binaries/Win64` path relatively, then execute `Game.exe` directly. Alternatively, you can click the `settings` button (or the button with the cogwheel icon in my launcher) and select `Open Game Data Directory`, after that, follow the path `game/Game/Binaries/Win64` and execute `Game.exe` directly.

After launching the game without using my launcher, please see if the problem still happens or not. If it still does, it's something I can't fix since it's a bug from the game itself, this regards to the game's devs studio, which is Amazing Seasun Games, and I have no relation to the studio.

Another thing I want to add is the PC client of the game wasn't intended to be run on Linux, so I don't think the devs will support you in this case. Can you try switching Wine versions or the Wine's graphics API (like using DXVK, for example)?

**Comment by RSerejo** [comment]
![image](https://github.com/Leayal/SnowBreakLauncher-Dotnet/assets/43152685/65faf01c-4114-42ab-8aa0-a05f19ea6f6e)

**Comment by Leayal** [comment]
Are you using Wine or Proton to start the game....?
From what I recall, Steam uses Proton to start non-Linux-native games.

How to configure Proton to work on this game is out of my reach as I also don't know how to...
In fact, I have very little knowledge about both Wine and Proton.

**Comment by Wildikdog** [comment]
The game requires `-FeatureLevelES31 -ChannelID=seasun -userdir='/path/to/game/Snow/data/game/'` to be put in the launch options to run from Steam or Lutris, as far as the vibrating login page that is Wine version related I think, just gotta test a few different versions of Wine/Proton to see which ones don't cause it.

**Comment by RSerejo** [comment]
> The game requires `-FeatureLevelES31 -ChannelID=seasun -userdir='/path/to/game/Snow/data/game/'` to be put in the launch options to run from Steam or Lutris, as far as the vibrating login page that is Wine version related I think, just gotta test a few different versions of Wine/Proton to see which ones don't cause it.

the game open with that launch options but the vibrating login page still persist

**Comment by Leayal** [comment]
> the game open with that launch options but the vibrating login page still persist

My 2cent (or may be educated guess) is it's either Wine/Proton or your OS that caused the glitches (I'm not saying your OS is glitchy, though). My launcher simply handles downloading files and, on Linux, tells [WINE](https://wiki.winehq.org/Main_Page) to start the game itself. After that point, everything is on the shoulder of WINE and the OS.

Unless my launcher downloads wrong files, which I verified that it shouldn't be the case here.
Everything wrong happening in-game shouldn't be caused by this launcher.
The proof was that even though you don't use my launcher to launch the game, the problem still occured.

Also, I think that if you can login just fine and the after-login gameplay doesn't have any issues, you can ignore the vibrating login form. Unless you change the path in `-userdir` param, delete something of the game or doing something wrong, you only need to login once as the game will remember your account.

Another thing I want to add is that this game has anti-cheat with it.
Anything that may trigger the anti-cheat mechanism will be the risk. As I said before, running the game on Linux isn't official supported by devs. Playing the game on Linux via WINE (or any layers/emulations) bear a risk that the Wine/Proton rubbed the anti-cheat the wrong way and trigger false-positive ban.
tl;dr: Play the game on Linux at your own risk. I'm not trying to ruin your mood, but knowing this is better than not.

**Comment by Leayal** [comment]
I'll close this issue as the graphics glitch is not caused by the launcher.

**Comment by sykhangdha** [comment]
Understand this issue is closed but the game actually works perfectly fine on linux if anyone is wondering(although ban is still possible obviously). Use "winecfg" in the terminal and in "Graphics" checkmark "Emulate Virtual Desktop" and enter your resolution. Initially the game loads and will flicker when the login screen pops-up BUT you can still click and enter your email or password(or login using google/facebook). After the game runs perfectly fine with no flickering.
![Screenshot from 2023-12-09 22-30-49](https://github.com/Leayal/SnowBreakLauncher-Dotnet/assets/31758072/6c7c1860-a4f6-4382-92da-559fb6b3e605)
